### PR TITLE
[2.0.2] en: source level filter support

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added, or deleted? (Required)

Add information about configuring binlog event filter in the source configuration file.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/563
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/source-configuration-file.md [hunk 1]
```diff
@@ -37,6 +37,16 @@ from:
 #   interval: 3600
 #   expires: 0
 #   remain-space: 15
+
+# Configure binlog event filters. (New in DM v2.0.2)
```
Comment (qiancai) [inline en/source-configuration-file.md:41]: Suggest using the same statement to describe the same change. 

```suggestion
# Starting from DM v2.0.2, you can configure binlog event filters in the source configuration file.

```

File: en/key-features.md [hunk 2]
```diff
@@ -235,7 +235,8 @@ Binlog event filter is a more fine-grained filtering rule than the block and all
 
 > **Note:**
 >
-> If the same table matches multiple rules, these rules are applied in order and the block list has priority over the allow list. This means if both the `Ignore` and `Do` rules are applied to a single table, the `Ignore` rule takes effect.
+> - If the same table matches multiple rules, these rules are applied in order and the block list has priority over the allow list. This means if both the `Ignore` and `Do` rules are applied to a single table, the `Ignore` rule takes effect.
```
Comment (qiancai) [inline en/key-features.md:238]: There is no need to highlight that it is a single table.
```suggestion
> - If the same table matches multiple rules, these rules are applied in order and the block list has priority over the allow list. This means if both the `Ignore` and `Do` rules are applied to a table, the `Ignore` rule takes effect.
```

**Review by qiancai — COMMENTED** [review]
@CharLotteiu  I've added two suggestions. PTAL. Thanks.

File: en/source-configuration-file.md (hunk 1 above)
Comment (CharLotteiu) [reply en/source-configuration-file.md:41]: The sentence here is a code comment. Generally, a code comment tells the user what the following codes are used for, so I change the perspective of this sentence.

Comment (qiancai) [reply en/source-configuration-file.md:41]: LGTM

**Pull request merged**